.github/workflows/release.yaml

```
@@ -1,95 +1,42 @@
-      - uses: actions/download-artifact@v8
-        with:
-          path: assets
-          pattern: binaries-*
-          merge-multiple: true
-
-      - name: Publish release
+      - name: Record unsigned upload location
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VERSION: ${{ github.ref_name }}
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)